### DISCUSSION TITLE
Food Log Ideas (and accolades!)

### DISCUSSION THREAD

**Discussion by pjfarr** [jlai/fitness-dashboard#19] [Ideas]
What a great project! Have been using it since the official Fitbit page vanished. Many things are a big improvement over the original UI, for instance the checkboxes in the food log that make it easier to delete or move several items at once. That's something I suggested to Fitbit years ago that they of course ignored.

Some ideas:
- Put checkboxes on the food list as well so that several items can be checked off then added to the log at once
- Checking off multiple items on the food list or items that have already been logged and having an option to create a custom meal out of those items or add them to existing custom meals
- I'd like to see the day's total calories and macros etc. moved to the _top_ of the log if possible, rather than having to scroll to the bottom to see it

Keep up the good work! It's greatly appreciated!

**Comment by jlai** [comment]
Thanks for the suggestions (and accolades)! I've added a "create meal" button for selected food logs. I've added a setting (https://dashboard.exercise.quest/settings, under the upper-right menu) to let you show the totals row at the top, bottom, or both.

Logging multiple foods — some kind of quick-logging feature — is a bit more complex design-wise. Will need to think about it.
